### PR TITLE
en,zh: Add doc for IPv6

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]

File: en/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -642,6 +642,20 @@ spec:
 
 See [Kubernetes Service Documentation](https://kubernetes.io/docs/concepts/services-networking/service/) to know more about the features of Service and what LoadBalancer in the cloud platform supports.
 
+### IPv6 Support
+
```
Comment (ran-huang) [inline en/configure-a-tidb-cluster.md:646]: Do you want to mention that "The TiDB version must be >= v6.5.1"?

File: zh/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -617,6 +617,20 @@ spec:
 
 访问 [Kubernetes Service 文档](https://kubernetes.io/docs/concepts/services-networking/service/)，了解更多 Service 特性以及云平台 Load Balancer 支持。
 
+### IPv6 支持
+
+自 TiDB 集群 v6.5.1 起，v1.4.3 或以上版本的 TiDB Operator 支持通过配置 `spec.preferIPv6` 为 `ture` 来部署监听 IPv6 地址的 TiDB 集群。
```
Comment (ran-huang) [inline zh/configure-a-tidb-cluster.md:622]:
```suggestion
TiDB 自 v6.5.1 起支持使用 IPv6 地址进行所有网络连接。如果你使用 v1.4.3 或以上版本的 TiDB Operator 部署 TiDB，你可以通过配置 `spec.preferIPv6` 为 `ture` 来部署监听 IPv6 地址的 TiDB 集群。
```

File: en/configure-a-tidb-cluster.md [hunk 3]
```diff
@@ -642,6 +642,20 @@ spec:
...
+Starting from TiDB v6.5.1, TiDB clusters that listen on IPv6 addresses can be deployed by TiDB Operator whose version is larger or equal than v1.4.3 with configuring `spec.preferIPv6` to `true`.
```
Comment (ran-huang) [inline en/configure-a-tidb-cluster.md:647]:
```suggestion
Starting v6.5.1, TiDB supports using IPv6 addresses for all network connections. If you deploy TiDB using TiDB Operator v1.4.3 or later versions, you can enable the TiDB cluster to listen on IPv6 addresses by configuring `spec.preferIPv6` to `true`.
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**